### ISSUE TITLE
JAXB Error while trying to run GUI for the first time

### ISSUE THREAD

**Issue by rutym** [matsim-org/matsim-code-examples#176]
Hi all.
I've downloaded the newest version of MATSIM (10.1) and I'm trying to run some of the example simulations, but I get the following error:
```
ERROR MatsimRuntimeModifications$1:41 Getting uncaught Exception in Thread main
java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
	at org.matsim.core.controler.corelisteners.DumpDataAtEndImpl.dumpLanes(DumpDataAtEndImpl.java:199)
	at org.matsim.core.controler.corelisteners.DumpDataAtEndImpl.notifyShutdown(DumpDataAtEndImpl.java:130)
	at org.matsim.core.controler.ControlerListenerManagerImpl.fireControlerShutdownEvent(ControlerListenerManagerImpl.java:122)
	at org.matsim.core.controler.AbstractController$1.shutdown(AbstractController.java:89)
	at org.matsim.core.controler.MatsimRuntimeModifications.run(MatsimRuntimeModifications.java:66)
	at org.matsim.core.controler.MatsimRuntimeModifications.run(MatsimRuntimeModifications.java:31)
	at org.matsim.core.controler.AbstractController.run(AbstractController.java:92)
	at org.matsim.core.controler.NewControler.run(NewControler.java:75)
	at org.matsim.core.controler.Controler.run(Controler.java:202)
	at org.matsim.run.Controler.run(Controler.java:56)
	at org.matsim.run.Controler.main(Controler.java:60)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.JAXBException
```
This happens in a few of the simulations, and I'm not sure what the problem is.
Has anyone encountered  such an error when running some of the examples for the first time?

**Comment by mrieser** [comment]
Likely you try to run MATSim with Java 9 or newer, which no longer contain a library required by MATSim (for details, e.g. see #31). Best is to run MATSim 10.1 with Java 8, or to add the JAXB libraries manually.

**Comment by rutym** [comment]
Thanks, that is indeed what I eventually find out, when looking in Stack Overflow.
Unfortunately, I can run only Java 11.
I tried to add manually the JAXB libraries manually but it didn't work. I tried adding it under the 'libs' directory, but it didn't work. Can it be I should change its name?

**Comment by mrieser** [comment]
you need to add the additional libraries to the classpath, just adding to the libs-directory is sadly not enough. This means that you must run MATSim from the command line, something like this:

    java -Xmx2g -cp matsim-0.10.1.jar:libs/jaxb.jar org.matsim.run.Controler config.xml

So you must list the matsim.jar and also all of the jaxb.jars after the `-cp` (which stands for class path). If you are on Windows, use `;` (semicolon) to separate the different jars, on Linux and macOS you have to use `:`.
(not sure how the jar-files are named exactly, please check when entering the command)

**Comment by rutym** [comment]
Thanks! This worked. I had to add all 3 required dependencies:

**Comment by rutym** [comment]
Thanks! This worked. I had to download the following 3 libraries:
- jaxb-api
- jaxb-core
- jaxb-impl
Note that in the main jar's maven file, there's a commented out area that define these dependencies. Before downloading the packages and running the commend you suggested, I first tried to comment-in these dependencies. This didn't work (I assume the jar files are missing from the 'libs' directory. It is worth adding them to there.
